test/spec/oidc/util/handleOAuthResponse.ts

```diff
@@ -174,6 +170,18 @@ describe('handleOAuthResponse', () => {
           }
           expect(errorThrown).toBe(true);
         });
+        it('throws if dpop=true and token_type is not DPoP', async () => {
+          let errorThrown = false;
+          try {
+            await handleOAuthResponse(sdk, { responseType: ['token', 'id_token'], dpop: true }, { }, undefined as unknown as CustomUrls);
```

Comment:
What's the double `as` for?

Reply:
Copied from test(s) above. Doesn't seem to have any affect

Reply:
I've run into this before where TS complains about a direct cast between "incompatible" types, so you have to cast it to `any` or `unknown` in between. 🤷🏼‍♀️